common-template/src/main/resources/templates/email/Lifecycle/retiringLifecycleInstantEmailBody.html

```
@@ -39,15 +42,15 @@
 {/if}
 
 {! RHEL 10 application streams life cycle section !}
-{#if rhel10SystemsCount > 0}
+{#if rhel10Data.count.or(-1) >= 0}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)